### PR TITLE
Added edit your profile

### PR DESCRIPTION
**Pull request by Mollysaurusen**
Added a basic function to edit your profile

### CODE REVIEW

File: endreprofil.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * TODO: Make the new crime script working again, but better!
+ */
```
Comment (Worzaii) [inline endreprofil.php:4]: Why is this comment about the crime script here? "Todo"'s are plans on what to do next.

File: endreprofil.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * TODO: Make the new crime script working again, but better!
+ */
+
+use UserObject\EndreProfil\EndreProfil;
+
+include("core.php");
+include_once '/classes/MinSide.php';
```
Comment (Worzaii) [inline endreprofil.php:9]: Why is this class included? Is it used in any way?

File: EndreProfil.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace UserObject\EndreProfil;
+
+use PDO;
+use UserObject\User;
+
+class EndreProfil
```
Comment (Worzaii) [inline EndreProfil.php:8]: Why isn't this extending "mainclass" to inherit used functions such as execute?

File: EndreProfil.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    public function __construct(User $user, PDO $database)
+    {
+        $this->user = $user;
+        $this->database = $database;
+        $this->execute();
+    }
+    
+    private function execute()
+    {
+        if ($write = canUseFunction(1, 1)) {
```
Comment (Worzaii) [inline EndreProfil.php:23]: This function will hinder the user from using the function if jailed or bunkered. Either set values to 0 and 0 unless the intention is to not allow changing a profile while either option is active, or simply do "readyEndreProfil" without the if clause.

**Review by Worzaii — CHANGES_REQUESTED** [review]
A few tips and guidelines. One commit seem to look exactly alike one other commit. Check reviews.

File: endreprofil.php (hunk 2 above)
Comment (Mollysaurusen) [reply endreprofil.php:9]: > Why is this class included? Is it used in any way?

You can replace it whit

 include_once __DIR__ . '/classes/EndreProfil.php';

**Comment by Worzaii** [comment]
No reply or changes made, closing request to pull.

**Pull request closed without merging**